src/css/blocks/card.css

```diff
@@ -0,0 +1,77 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  color: #616070;
+  --color-dark-blue: #060b35;
+}
+
+.card__image {
+  padding: 0 3px;
+  width: 100%;
+  height: 134px;
+  margin-bottom: 38px;
+  object-fit: contain;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--color-dark-blue);
+  margin-bottom: 4px;
+}
+
+.card__code {
+  margin-bottom: 16px;
+  line-height: 14px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  text-align: right;
+  color: var(--color-dark-blue);
+}
+
+.card__price-section {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--color-dark-blue);
+}
+
+.card__buy-link {
+  display: block;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 14px;
+  text-align: center;
+  color: white;
+  padding: 11px 0;
+  background: #00acdc;
```

Comment:
set specific property, it's background color, there is no other background values here

Reply:
Done.